### PR TITLE
chore: [MICROBA-1498] Add bulk email component structure

### PR DESCRIPTION
**Pull request by Tj-Tracy**
Add component structure for the bulk email tool and a readme rework.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,76 +1,51 @@
-|Build Status| |Codecov| |license|
+|Build Status| |Codecov| |license| |semantic-release| ## As many of these as you'd like to include, or others if you wish!
 
 frontend-app-communications
-=================================
+==============================
 
-Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
+Please tag **@edx/aperture-eng** on any PRs or issues.  Thanks!
```
Comment (justinhynes) [inline README.rst:6]: Is this supposed to be our GitHub team name? I think our team name is `edx-aperture`

File: README.rst [hunk 2]
```diff
@@ -1,76 +1,51 @@
...
-The dev server is running at `http://localhost:8080 <http://localhost:8080>`_ or whatever port you setup.
+   .. code-block::
 
-Making Your New Project's README File
--------------------------------------
+      npm i
 
-Move the file ``README-template-frontend-app.rst`` to your project's ``README.rst`` file. Please fill out all
-the sections - this helps out all developers understand your MFE, how to install it, and how to use it.
+3. start the devserver. The app will be running at `localhost:8080`, or whatever port you change it too.
```
Comment (justinhynes) [inline README.rst:33]: Should we settle on a dedicated port now? If not, should we make a subtask to make sure we don't forget to do this? That way we won't have collisions with other MFE's once they get spun up?

File: README.rst (hunk 1 above)
Comment (Tj-Tracy) [reply README.rst:6]: honestly I have no clue and took a guess!

File: README.rst (hunk 2 above)
Comment (justinhynes) [reply README.rst:33]: I thought I saw a list somewhere of the ports used by various MFEs... trying to find that now. I know I stumbled across it when we were creating the learner-record over the summer.

Comment (justinhynes) [reply README.rst:33]: Ah, yes, here are a few I know that are in use right now...
```
    'localhost:1997',  # frontend-app-account
    'localhost:1976',  # frontend-app-program-console
    'localhost:1994',  # frontend-app-gradebook
    'localhost:2000',  # frontend-app-learning
    'localhost:2001',  # frontend-app-course-authoring
    'localhost:3001',  # frontend-app-library-authoring
```

The learner-record MFE uses `1990` as well. Som anything that isn't being used?

**Review by justinhynes — APPROVED** [review]
LGTM. Approved knowing/assuming that the red errors/warnings yelling at us will go away as this is just a preliminary starting point/framing PR.

File: README.rst (hunk 2 above)
Comment (Tj-Tracy) [reply README.rst:33]: `1984` it is

**Pull request merged**